osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySliderBall.cs

```diff
@@ -30,8 +34,10 @@ public LegacySliderBall(Drawable animationContent, ISkin skin)
         }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(DrawableHitObject dho)
```

Comment:
You can also move this to a `[Resolved]` property and rename it to `DrawableObject` as per

https://github.com/ppy/osu/blob/f8830c6850128456266c82de83273204f8b74ac0/osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyMainCirclePiece.cs#L42-L43